{"name": "test", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'", "trace": "@pytest.mark.login\n    def test():\n>       driver = webdriver.Edge(executable_path=edge_driver_path, options=edge_options)\nE       TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n\ntests\\test_sample.py:24: TypeError"}, "start": 1717057708284, "stop": 1717057708284, "uuid": "5116b991-974b-4bb5-8f9b-d4d11b976117", "historyId": "54d0412a5319a1fc0367fbeebdbbe70d", "testCaseId": "54d0412a5319a1fc0367fbeebdbbe70d", "fullName": "tests.test_sample#test", "labels": [{"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_sample"}, {"name": "host", "value": "INHNJLPT-984680"}, {"name": "thread", "value": "28140-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_sample"}]}